platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/state/service/PbjConverterTest.java

```diff
@@ -58,7 +58,7 @@ class PbjConverterTest {
 
     public static final NodeId NODE_ID_1 = new NodeId(1);
     public static final NodeId NODE_ID_2 = new NodeId(2);
-    private final Random random = new Random();
+    private static final Random random = new Random();
```

Comment:
I suggest not using a static random, it can be difficult to recreate an issue if it occurs. I suggest using a `Randotron` per test which prints a seed that was used. It also has some additional functionalities